webapp/src/websocket_handlers.ts

```diff
@@ -473,3 +476,53 @@ export function handleHostScreenOff(store: Store, ev: WebSocketMessage<{ channel
 
     client.unshareScreen();
 }
+
+export function handleHostUnraiseHand(store: Store, ev: WebSocketMessage<{
+    call_id: string,
+    channel_id: string,
+    session_id: string,
+    host_id: string
+}>) {
+    const channelID = ev.data.channel_id;
+    const client = getCallsClient();
+    if (!client || client?.channelID !== channelID) {
+        return;
+    }
+
+    const sessionID = client.getSessionID();
+    if (ev.data.session_id !== sessionID) {
+        return;
+    }
+
+    client.unraiseHand();
+
+    const profiles = profilesInCurrentCallMap(store.getState());
+    const displayName = getUserDisplayName(profiles[ev.data.host_id]);
+
+    const hostNotification: HostControlNotification = {
+        type: HostControlNotificationType.UnraisedHand,
+        callID: ev.data.call_id,
+        notificationID: generateId(),
+        displayName,
+    };
+
+    // Put the notification on the end of the event loop so that unraiseHand can be processed before
+    // we continue. This prevents the "raised hand" and "host has lowered your hand" reaction chips
+    // from being shown at the same time.
+    setTimeout(() => {
+        store.dispatch({
+            type: HOST_CONTROL_NOTIFICATION,
+            data: hostNotification,
+        });
+    }, 0);
```

Comment:
Not sure I fully understood. Is this conflicting with a future websocket event?

Reply:
No, it just that if you send it right now there will be two reaction chips at the same time: the "you are raising your hand" + the hostControlNotification. That was causing a jump because the hostControlNotifcation would animate in.